atlite/convert.py

```diff
@@ -493,6 +493,76 @@ def wind(cutout, turbine, smooth=False, **params):
     )
 
 
+# irradiation
+def convert_irradiation(
+    ds, orientation, irradiation="total", trigon_model="simple", clearsky_model="simple"
```

Comment:
I think I just did exactly what the `pv` and `convert_pv` methods do, since I wasn't sure how you wanted defaults like this to be handled. Is there something I'm missing, or does the `convert_pv` method also have a redundant default `clearsky_model="simple"`?

Reply:
Ok, regarding your later comment of how the `clearsky_model` is used, I think it's indeed better to leave the default value `None` for both functions.